### ISSUE TITLE
Incorrect information about device function ARGS

### ISSUE THREAD

**Issue by sethherr** [particle-iot/docs#1109]
**TL;DR update the Cloud API docs** with the function argument information from the Device OS API Docs.

In [call a function](https://docs.particle.io/reference/device-cloud/api/#call-a-function) on the Cloud API docs the information about the ARG parameter reads:

> Function argument with a maximum length of 63 characters

<img width="510" alt="Screen Shot 2020-07-21 at 12 03 47 PM" src="https://user-images.githubusercontent.com/1235441/88095859-9d949400-cb4a-11ea-8b44-4c0bf6688197.png">

In the Device OS API Docs (e.g. [Argon Particle.function](https://docs.particle.io/reference/device-os/firmware/argon/#particle-function-)) it has different information:

> A cloud function is set up to take one argument of the String datatype. This argument length is limited to a max of 63 characters (prior to 0.8.0), 622 characters (since 0.8.0). The String is UTF-8 encoded.

<img width="502" alt="Screen Shot 2020-07-21 at 12 05 52 PM" src="https://user-images.githubusercontent.com/1235441/88096077-f401d280-cb4a-11ea-99e7-06c8a7c5dc15.png">

The arg parameter does appear to be limited to 622 characters now - which is great!

p.s. I would've submitted a PR to fix this, but it doesn't appear to be possible, I couldn't find the text in the repository.

**Comment by rickkas7** [comment]
Thanks for the feedback. You are correct and the string doesn't exist in the docs repository because it's auto-generated from comments in the source, so it has to be changed there.
